app/javascript/src/components/Dashboard/Notes/Note/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { Clock } from "neetoicons";
+import { Typography, Button, Avatar } from "neetoui";
+
+const Footer = ({ createdAt }) => (
+  <div className="flex flex-row items-center justify-between">
+    <Button
+      className="text-gray-400"
+      label="Getting Started"
+      style="secondary"
+    />
+    <div className="flex flex-row gap-2 text-gray-400">
```

Comment:
Why do we need to pass `text-gray-400` here?

Reply:
I wanted to give light gray color for this footer component.

Reply:
@chiragbagde https://www.loom.com/share/480e134db41446498eee7ae3155255ec

Reply:
When I click on this property in the Figma design for this challenge, it displays the CSS code as font-weight 400 with light grey.